api/v1alpha4/conversion.go

```diff
@@ -238,3 +243,11 @@ func Convert_v1beta1_MachineDeploymentTopology_To_v1alpha4_MachineDeploymentTopo
 	// MachineDeploymentTopology.FailureDomain has been added with v1beta1.
 	return autoConvert_v1beta1_MachineDeploymentTopology_To_v1alpha4_MachineDeploymentTopology(in, out, s)
 }
+
+func Convert_v1beta1_MachineDeploymentClass_To_v1alpha4_MachineDeploymentClass(in *clusterv1.MachineDeploymentClass, out *MachineDeploymentClass, s apiconversion.Scope) error {
+	return autoConvert_v1beta1_MachineDeploymentClass_To_v1alpha4_MachineDeploymentClass(in, out, s)
```

Comment:
nit: Would be good to add a comment (see above) that the MHC field has been added 

(same below)